example.env

```diff
@@ -0,0 +1,2 @@
+ADDRESS="127.0.0.1:9944"
```

Comment:
I believe these quotes are not required in this context.   I believe in the TypeScript dotenv these quotes will be included in the environment, not sure how rust behaves but its worth checking